charts/dapr/charts/dapr_placement/templates/dapr_placement_statefulset.yaml

```diff
@@ -253,6 +253,10 @@ spec:
       tolerations:
 {{ toYaml .Values.global.tolerations | indent 8 }}
 {{- end }}
+{{- if .Values.global.priorityClassName }}
```

Comment:
Thanks for finding this - mind adding this same diff for this file: `dapr_scheduler_statefulset.yaml`?

Reply:
It seems to be already present in the file `dapr_scheduler_statefulset.yaml`

https://github.com/m3nax/dapr/blob/24ba30bc4879eee63106aed877858f322f4ca223/charts/dapr/charts/dapr_scheduler/templates/dapr_scheduler_statefulset.yaml#L271-L274

Reply:
ah, yes I mixed up the indentation and spec and was thinking we needed this same diff for the scheduler, but we dont. thx, you are correct we are good there.